ITISFoundation/osparc-simcore
# 🐛 Fix aws checksum issue

## What do these changes do?
Add the sha256 checksum of an S3 object directly in the `Metadata` field instead of adding it via the `ChecksumSHA256` field. 
The reason why this is required is discussed in some detail in the issue listed below. 

## Related issue/s
- fixes https://github.com/ITISFoundation/osparc-simcore/issues/4885



## How to test
This was tested manually using an S3 test bucket on my local deployment. I believe all multipart uploads via storage to AWS S3 are currently broken by this, hence the multipart upload in the client e2e tests should actually test this. Hence, instead of introducing additional tests I think it is important, that I propagate the multipart upload in the public API to production, so that those tests will also run against AWS S3.



## DevOps Checklist
